src/services/lease-service/tests/routes/offers.test.ts

```diff
@@ -273,4 +273,32 @@ describe('offers', () => {
       expect(res.status).toBe(200)
     })
   })
+
+  describe('GET /offers/listing-id/:listingId', () => {
+    it('should return a list of offers', async () => {
+      jest
+        .spyOn(offerAdapter, 'getOffersByListingId')
+        .mockResolvedValueOnce({ ok: true, data: [factory.offer.build()] })
+
+      const res = await request(app.callback()).get(`/offers/listing-id/1`)
+
+      expect(res.status).toBe(200)
+      expect(res.body).toMatchObject({
+        content: expect.arrayContaining([
+          expect.objectContaining({ id: expect.any(Number) }),
+        ]),
+      })
+    })
+
+    it('should return an error if not found', async () => {
```

Comment:
Maybe the test name is what confuses me here, but I would expect an empty list and not error if there are no offers for a listing? 

Maybe rename the test to "should handle errors" or something like that?

Reply:
https://github.com/Bostads-AB-Mimer/onecore-leasing/pull/131/commits/647e9b308267405bd8d6fc09fa3b88a1d834f443

Yeah it was confusing 👍